Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -4310,21 +3007,31 @@ script:
     - contextPath: PaloAltoNetworksXDR.FeaturedField.fields
       description: String value that defines the new field.
       type: String
-  dockerimage: demisto/python3:3.10.8.36650
-  feed: false
+  - arguments:
+    - description: List of endpoint IDs. Supports comma separated list.
+      isArray: true
+      name: endpoint_ids
+    - description: Tag to add.
+      name: tag
+    description: Adds a tag to specified endpoint_ids
+    name: xdr-endpoint-tag-add
+  - arguments:
+    - description: List of endpoint IDs. Supports comma separated list.
+      isArray: true
+      name: endpoint_ids
+    - description: Tag to remove from specified endpoint_ids
+      name: tag
+    description: Removes a tag from specified endpoint_ids.
+    name: xdr-endpoint-tag-remove
+  dockerimage: demisto/python3:3.10.8.37233
   isfetch: true
-  longRunning: false
-  longRunningPort: false
   runonce: false
-  script: '-'
+  script: ''
   subtype: python3
   ismappable: true
   isremotesyncin: true
   isremotesyncout: true
   type: python
 tests:
 - Test XDR Playbook
-defaultmapperin: Cortex XDR - IR-mapper
-defaultclassifier: Cortex XDR Incident Handler - Classifier
-defaultmapperout: Cortex XDR - IR-out-mapper
```

Comment:
why were these removed?

Reply:
Was not removed by me. Possible issue during marketplace contribution process.

Reply:
I see will check it, and reverted this :)